Retool outreach email (#1)
dolthub/cold-marketing: 01-04-2023-version-control-your-retool-app.md

---

Subject: "Make Retool Better with Version Control"
Date: "2023-01-04"
ICP: "VP ENG, CTO, Devs"
Market: "Any company using Retool"

---

Hi First Name,

Retool is a powerful tool that allows low-code solution to create internal tools. I'm sure you are aware as a user :)

> What if you added a version controlled database to that process? You no longer have to worry about accidently making changes to your production database. Maybe today you work off of a copy or a backup. But how do you manage changes to your database?

Reviewer: ```suggestion
What if you added a version controlled database to your Retool applications? You no longer have to worry about accidentally making changes to your production database. Maybe today you work off of a copy or a backup. But how do you manage changes to your database?
```

> Dolt allows you to do just that. Commits, branches, and pull requests are built into Dolt. If you use it as a backing store for our Retool app, then you will get those features in your app automatically. 

Reviewer: ```suggestion
Dolt allows you to do just that. Commits, branches, and pull requests are built into Dolt. If you use Dolt as the database for your Retool app, then you will get those features in your app automatically. 
```
